src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double longerBase;
+    private final double shorterBase;
+    private final double height;
+
+    public IsoscelesTrapezoid(double longerBase, double shorterBase, double height, String color) {
+        super(color);
+        this.longerBase = longerBase;
+        this.shorterBase = shorterBase;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((shorterBase + longerBase) * height) / 2;
+    }
+
+    @Override
+    public void toDraw() {
```

Comment:
The method name 'toDraw' seems like a verb, which is not recommended for method names. Consider renaming it to something more noun-like, such as 'draw' to align with the checklist item about not using verbs for method names.